### DISCUSSION TITLE
Implement Lazy Loading

### DISCUSSION THREAD

**Discussion by routerino** [outline/outline#5528] [Ideas]
I've had complaints about [my blog](https://blog.gurucomputing.com.au/s/blog/doc/installing-outline-tXZO9ehoFV) that loads are slow, and this seems primarily due to my semi-regular use of animated gifs in the articles.

Is lazy loading on outline (or at least shared documentation from outline) feasible? It would help SEO a lot.

**Comment by tommoor** [comment]
Agreed that it’s slow - Outline is not a blogging platform so it’s not ideal for this, a lot of JS is loaded on first run but even given that it seems slow - perhaps are you hosted in Australia? Is the box underpowered? Are you using a CDN?

**Reply by routerino** [reply]
Yes hosted in australia, not using a CDN. It's a hobby blog, more for me to write up topics I'm interested in or interested in learning, not reaching for an actual reader base.

That said I suppose if it bothers me enough I can go back to exporting the markdown out of the drafts and using a dedicated static site generator.

**Reply by tommoor** [reply]
I'd imagine Aus is a big part of the problem, it's pretty isolated from the rest of the internet. Adding a CDN would help a lot.

**Comment by routerino** [comment]
Redone the blog to be exported to a static site generator now. As a side benefit, [updated my outline guide](https://blog.gurucomputing.com.au/Knowledgebases%20with%20Outline/Installing%20Outline%20Knowledgebase/). Should be much quicker now, even without a CDN
